yarn-project/noir-contracts/src/contracts/noir-aztec3/src/state_vars/set.nr

```diff
@@ -2,22 +2,20 @@ use dep::std;
 use crate::abi::Inputs;
 use crate::context::PrivateFunctionContext;
 use crate::notes::value_note::Note;
+use crate::notes::value_note::NOTE_LEN;
 use crate::types::point::Point;
-use crate::state_vars::note_getter_data::NoteGetterData;
-use crate::state_vars::note_getter_data::make_note_getter_data;
-use crate::state_vars::note_getter_data::note_getter_data_len;
+use crate::utils::arr_copy_slice;
 
 // oracles
 use crate::oracle::notes::notify_created_note;
 use crate::oracle::notes::notify_nullified_note;
 use crate::oracle::notes::get_notes_2_internal;
 
-
-fn get_2_notes(storage_slot: Field) -> (NoteGetterData, NoteGetterData) {
+fn get_2_notes(storage_slot: Field) -> (Note, Note) {
     let fields = get_notes_2_internal(storage_slot);
-    let data1 = make_note_getter_data(fields, 1);
-    let data2 = make_note_getter_data(fields, 1 + note_getter_data_len());
-    (data1, data2)
+    let note1 = Note::deserialize(arr_copy_slice(fields, [0; NOTE_LEN], 1));
+    let note2 = Note::deserialize(arr_copy_slice(fields, [0; NOTE_LEN],  1 + NOTE_LEN));
+    (note1, note2)
```

Comment:
No longer need `note_getter` helper since oracle only returns count and note preimages.